docs/UserGuide.md

```diff
@@ -158,7 +217,7 @@ Format: `edit NRIC [n/NAME] [i/NRIC] [g/GENDER] [d/DOB] [p/PHONE_NUMBER] [e/EMAI
 > * Refer to the [Parameter Details](#parameter-details) section for more information on the constraints for each parameter.
 
 Example:
-*  `edit S1234567A p/91234567 e/johndoe@example.com` Edits the phone number and email address of the patient with the NRIC`S1234567A`
+*  `edit S1234567A p/91234567 e/johndoe@example.com` will edit the phone number and email address of the patient with the NRIC`S1234567A`
 to `91234567` and `johndoe@example.com` respectively.
```

Comment:
Good idea of using modal auxiliary verb *will* to denote conditional consequence of action instead of simple present denoting habitual modality. 

gives the user a sense of control